src/common.rs

```diff
@@ -1091,6 +1097,21 @@ mod tests {
         assert_eq!(res.unwrap().1, expected);
     }
 
+    #[test]
+    fn multi_args_function() {
+        let qs = b"CONCAT(first_name, ' ', last_name)";
+        let qs1 = "CONCAT('%', 'abc', '%')";
+
+        let res = column_function(qs);
+        let expected = FunctionExpression::Concat(vec![
+            FieldDefinitionExpression::Col(Column::from("first_name")),
+            FieldDefinitionExpression::Value(FieldValueExpression::Literal(LiteralExpression::from(Literal::String(" ".to_string())))),
+            FieldDefinitionExpression::Col(Column::from("last_name")),
+        ]);
+        assert_eq!(res.unwrap().1, expected);
+        assert_eq!(format!("{}", column_function(qs1.as_bytes()).unwrap().1), qs1);
+    }
+
```

Comment:
Might be good to add an expected-to-fail test for an incorrectly bracketed variant here. Check that `res.is_err()` is true.

Reply:
Thank you for your instruction and advice. I've learned a lot.